docs/ecosystem/prod.md

```diff
@@ -26,3 +21,8 @@ Trivy is natively integrated into Harbor, no installation is needed. More info i
 You can use Kyverno to ensure and enforce that deployed workloads' images are scanned for vulnerabilities.
 
 👉 Get it at: <https://neonmirrors.net/post/2022-07/attesting-image-scans-kyverno>
+
+### Devopstales Trivy Operator (Community)
```

Comment:
do we want to mention community-created operators here? Just wondering about the goal

Reply:
I thought the goal is to showcase the ecosystem around trivy, no? do you think against community integrations or just specifically this one?

Reply:
I am just thinking specifically this one since we want to lead people to the official trivy-operator; listing both might lead to confusion even if one is official and the other one created an maintained by the community

Reply:
ok I'll remove it for now